servicetests/test_cases/mqtt3_fleet_provisioning_cfg.json

```diff
@@ -1,34 +0,0 @@
-{
```

Comment:
It looks like we only have unit tests for V2 service clients, I feel in that case we should keep the mqtt3 ones? 

Feel sorry that I brought that up, and I didn't realize the `sdk/serviceclients` was only for V2 client. Though we don't run the mqtt3 service client anymore, We should still have some testing against the Mqtt3 ones.